src/Microsoft.CodeAnalysis.NetAnalyzers/src/RulesMissingDocumentation.md

```diff
@@ -2,7 +2,3 @@
 
 Rule ID | Missing Help Link | Title |
 --------|-------------------|-------|
-CA1873 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1873> | Avoid potentially expensive logging |
-CA1874 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1874> | Use 'Regex.IsMatch' |
-CA1875 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1875> | Use 'Regex.Count' |
-CA2023 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2023> | Invalid braces in message template |
```

Comment:
@ViktorHofer We're seeing this file churn when we build locally, is this a change that should be checked in?

Reply:
I got the same thing too on my machine, so definitely some line endings that need fixed up.

Reply:
Yes I think this needs to get checked-in.

Reply:
All these rules are actually missing documentation.